src/GovUk.Education.ExploreEducationStatistics.Admin/Services/BootstrapUsersService.cs

```diff
@@ -1,61 +1,46 @@
 #nullable enable
-using GovUk.Education.ExploreEducationStatistics.Admin.Database;
-using GovUk.Education.ExploreEducationStatistics.Admin.Models;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Models.GlobalRoles;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services;
 
-public class BootstrapUsersService
+public class BootstrapUsersService(IConfiguration configuration, ContentDbContext contentDbContext)
 {
-    private readonly IConfiguration _configuration;
-    private readonly UsersAndRolesDbContext _usersAndRolesDbContext;
-    private readonly ContentDbContext _contentDbContext;
-
-    public BootstrapUsersService(
-        IConfiguration configuration,
-        UsersAndRolesDbContext usersAndRolesDbContext,
-        ContentDbContext contentDbContext
-    )
-    {
-        _configuration = configuration;
-        _usersAndRolesDbContext = usersAndRolesDbContext;
-        _contentDbContext = contentDbContext;
-    }
-
     /**
      * Add any bootstrapping BAU users that we have specified on startup.
      */
     public void AddBootstrapUsers()
     {
-        var bauBootstrapUserEmailAddresses = _configuration
+        var bauBootstrapUserEmailAddresses = configuration
```

Comment:
There's a warning about 'Possible multiple enumeration' of `bauBootstrapUserEmailAddresses` here, so could add `ToArray` or `ToList` to this.